tools/osbuild-manifest

```diff
@@ -91,6 +104,8 @@ def main():
     parser = argparse.ArgumentParser(description="Manifest operations")
     parser.add_argument("--json", action="store_true",
                         help="output results in JSON format")
+    parser.add_argument("--store", metavar="DIRECTORY", default=None,
+                        help="directory where objects are stored in")
```

Comment:
Please mention that the default is `.osbuild` in the help text.

Reply:
But it isn't ;) At least not in that tool. Maybe it should. I always though that was a bit magic and specifically for `osbuild-manitfest` I wanted the store and the checking if objects exist to optional.
What do you think?

Reply:
Ah! I didn't realize. Your logic is flawless :wink: